cores/migrations/20240725012639_MakeCategoryCategoryNonNullable.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Add migration script here
+ALTER TABLE category
+ALTER COLUMN category
+SET
+  NOT NULL;
```

Comment:
No reason to have a category row without a category string, so set it to not null